src/components/App.js

```diff
@@ -6,10 +6,8 @@ import Checkout from './Checkout/Checkout';
 import LogIn from './LogIn/LogIn';
 import Navbar from './Navbar/Navbar';
 import SignUp from './SignUp/SignUp';
-import ProductDetail from './Product/ProductDetail';
```

Comment:
So correct me if i'm wrong. You removed ProductDetail, ProductDetailImages and Specifications components and added a  composite Product component which basically has all the info.